lib/parts.lua

```diff
@@ -63,6 +63,12 @@ function parts.init(soundEngine)
     ['is_nervous'] = false
   }
 
+  -- chords
```

Comment:
You can use `SoundEngine:getSetParam(id, name)` (to get a single-param setter) or `SoundEngine:setSynthParams(id, synthParams)` (to set more than 1 param at a time, where `synthParams` is a dict-table) if you like, too.

Reply:
done!